charts/brig/values.yaml

```
@@ -13,7 +13,7 @@ resources:
     memory: 512Mi
     cpu: 500m
 config:
-  logLevel: Info
+  logLevel: Debug
```

```suggestion
  logLevel: Info
```
I think it's fine if the default is still `Info`, we can override it per installation